src/main/java/org/apache/cassandra/distributed/api/CompleteQueryResult.java

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
this isn't a new file but renamed `QueryResult` to this and made `QueryResult` a interface.